Macros.h

```diff
@@ -142,4 +151,32 @@ static inline unsigned long long saturatingSub(unsigned long long a, unsigned lo
    return a > b ? a - b : 0;
 }
 
+#ifdef HAVE_ALIGNAS
+#include <stdalign.h>
+
+#define ELF_NOTE_DLOPEN_OWNER "FDO"
+#define ELF_NOTE_DLOPEN_TYPE  UINT32_C(0x407c0c0a)
+
+#define DECLARE_ELF_NOTE_DLOPEN(content) \
+   IGNORE_W11EXTENSIONS_BEGIN                                                                      \
+   __attribute__((used, section(".note.dlopen"))) alignas(sizeof(uint32_t)) static const struct {  \
```

Comment:
```suggestion
   __attribute__((packed, used, section(".note.dlopen"))) static const struct {                    \
```

Just byte-align ourselves and round up with some little magic later

Reply:
Another problem: C99 doesn't support anonymous struct yet.

Reply:
> Another problem: C99 doesn't support anonymous struct yet.

The workaround for that is kinda easy though, too …